### PR TITLE
DBZ-3167 Removed column.blacklist, cleaned up config definition

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-3167

### CODE REVIEW

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java [hunk 1]
```diff
@@ -472,10 +441,6 @@ public OracleVersion getOracleVersion() {
         return oracleVersion;
     }
 
-    public Tables.ColumnNameFilter getColumnFilter() {
```
Comment (gunnarmorling) [inline debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java:475]: How does the column filter work then now? Is it inherited from the framework? Is there a test for the Oracle connector for this?

**Review by gunnarmorling — COMMENTED** [review]
One question inline.

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java (hunk 1 above)
Comment (Naros) [reply debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java:475]: Yes the filter is part of the common framework; however I do need to add a test for this.  I thought this was tested in the core repository but looking at the connectors there they each have at least one test for this so I'll add one here for consistency.

**Comment by gunnarmorling** [comment]
Yes, we should have connector-specific tests for such shared functionality.
Ideally, we'd have a notion of a TCK which can be run against all the
(relational) connectors, but we're not there yet.

>

**Comment by Naros** [comment]
@gunnarmorling Tests added.

**Comment by Naros** [comment]
I need to add `column.include/exclude.list` to the docs as those aren't present.

File: debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OracleConnectorFilterIT.java [hunk 2]
```diff
@@ -453,6 +477,140 @@ private void shouldApplySchemaAndTableExclusionsConfiguration(boolean useLegacyO
         }
     }
 
+    private void shouldApplyColumnInclusionsConfiguration(boolean useLegacyOption) throws Exception {
+        Field option = OracleConnectorConfig.COLUMN_INCLUDE_LIST;
+        if (useLegacyOption) {
```
Comment (gunnarmorling) [inline debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OracleConnectorFilterIT.java:482]: So if we never documented these legacy option names as per what your say, I'd suggest let's simply remove them altogether. Chances anyone used them are really low then, and given the connector is incubating, the remaining risk seems acceptable to me.

Comment (Naros) [reply debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OracleConnectorFilterIT.java]: Fixed, tests now only use the include/exclude option names for column filtering.

**Review by gunnarmorling — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Awesome; pretty cool to see how we become such features essentially "for free" by means of the framework 🎉 .

**Pull request merged**